packages/x-data-grid-pro/src/hooks/features/rowReorder/useGridRowReorder.tsx

```diff
@@ -63,6 +64,16 @@ export const useGridRowReorder = (
   const ownerState = { classes: props.classes };
   const classes = useUtilityClasses(ownerState);
   const [dragRowId, setDragRowId] = React.useState<GridRowId>('');
+  const rows = gridExpandedSortedRowEntriesSelector(apiRef);
```

Comment:
Using a selector directly in the React component might yield bugs since it'll not ensure proper reactivity.
Wrapping the selector with `useGridSelector` is preferred as it ensures to re-render the component when the state value updates.